model/post_list.go

```diff
@@ -25,6 +25,15 @@ func NewPostList() *PostList {
 	}
 }
 
+func (o *PostList) Clone() *PostList {
+	return &PostList{
+		Order:      o.Order,
```

Comment:
I think we also need to copy Order and Posts here. I'm pretty sure they're stored as something like pointers that aren't copied.